test/unit_tests/wrappers/test_openwhisk_traces.js

```diff
@@ -46,9 +46,9 @@ describe('openwhiskWrapper tracer tests', () => {
             'addException'
         );
 
-        this.postTraceStub = sinon.stub(
+        this.sendTraceStub = sinon.stub(
```

Comment:
why is this change required? did tests failed without it?

Reply:
Yeah.
actually the entire test seems to be buggy.
TraceObj is not being initialised, therefore - no runner is being set (look at addRunner func), and the trace is will not be posted from now on  (which is a GOOD behavior) 
will fix that in the future.  @ranrib please approve.